CHANGELOG.md

```diff
@@ -2317,6 +2317,8 @@ v0.24.0 (2022-04-07)
 
 ### Features
 
+- Adding more option to enable required mongo stats to monitor. (@Nitesh-vaidyanath)
```

Comment:
```suggestion
- Adding more option to enable required MongoDB stats to monitor. (@Nitesh-vaidyanath)
```

We also need to correct the MongoDB name to match their product name/capitalization